### PR TITLE
Added context menu

### PR DESCRIPTION
**Pull request by StijnOostdam**

### CODE REVIEW

File: Winleafs.Wpf/Views/MainWindows/MostUsedEffectUserControl.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+namespace Winleafs.Wpf.Views.MainWindows
+{
+    /// <summary>
+    /// Interaction logic for MostUsedEffectUserControl.xaml
+    /// </summary>
+    public partial class MostUsedEffectUserControl : UserControl
+    {
+        private TaskbarIcon _parent;
+        private string _effectName;
```
Comment (bartdebever) [inline Winleafs.Wpf/Views/MainWindows/MostUsedEffectUserControl.xaml.cs]: Readonly

File: Winleafs.Wpf/Views/MainWindows/TaskbarIcon.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+            SetOverride();
+        }
+
+        private void SetOverride()
+        {
+            _parent.OverrideScheduleUserControl.SetOverride(_selectedEffect, Brightness);
+        }
+
+        private void BrightnessSlider_DragCompleted(object sender, System.Windows.Controls.Primitives.DragCompletedEventArgs e)
```
Comment (bartdebever) [inline Winleafs.Wpf/Views/MainWindows/TaskbarIcon.xaml.cs]: `using`  if possible

File: Winleafs.Wpf/Views/MainWindows/TaskbarIcon.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+            _parent.OverrideScheduleUserControl.StopOverride();
+
+            BuildMostUsedEfectList();
+        }
+
+        private void BuildMostUsedEfectList()
+        {
+            MostUsedEffectList.Children.Clear();
+
+            var mostUsedEffects = UserSettings.Settings.ActiveDevice.EffectCounter.Take(5).ToList();
```
Comment (bartdebever) [inline Winleafs.Wpf/Views/MainWindows/TaskbarIcon.xaml.cs]: Rather put the 5 in a constant

**Comment by StijnOostdam** [comment]
Check for self: what happens if you add a color and it is in the top 5 selected effects and then you delete it?

**Comment by StijnOostdam** [comment]
Misclick whew

**Pull request merged**